Create ui-c8y-1018-503-86-missing-translation-in-ui-6210.md (#2057)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1018-503-86-missing-translation-in-ui-6210.md

---
date: ""

> title: missing translation in UI (#6210)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Add missing translations for device management UI
```

> title: Provided translations for default roles

Reviewer: ```suggestion
title: Provided translations for default roles labels
```

> title: Provided translations for default global roles (#6210)

Reviewer 2: ```suggestion
title: Provided translations for default global roles in Japanese
```

Reviewer 3: translation being provided for all languages, so will not apply this suggestion

> title: Provided translations for default global roles (#6210)

Reviewer: ```suggestion
title: Provided translations for global roles labels
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-0UgqXH1Ys
    label: Administration
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-58187
version: 1018.503.86
---

> missing translation in UI (#6210)

Author: The Claude AI suggested to use the following body:
```suggestion
Some translations were missing in the device management UI, leading to untranslated text being displayed to users. The missing translations have now been added, so that all text in the device management UI is properly translated based on the user's language settings. This change improves the user experience and usability of the device management UI for non-English speaking users.
```

> The labels for the following roles are now translated. Global roles: "admins", "business", "devices". Inventory roles: "Manager".

Reviewer: ```suggestion
The labels for the following default roles are now translated. Global roles: "admins", "business", "devices". Inventory roles: "Manager".
```

> Provided translations for following global roles: "admins", "business", "devices", "readers". Also translated following inventory roles: "Manager". (#6210)

Reviewer 2: ```suggestion
Global roles: "admins", "business", "devices". Inventory roles: "Manager" are now translated to Japanese.
```

Reviewer 3: translation being provided for all languages, so will not apply this suggestion

Reviewer: @Airdroplet I updated the suggestion, please check.
